### ISSUE TITLE
Jaw Contracture

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6108]
**Preferred term label:** Jaw Contracture

**Synonyms**

**Definition (free text, please give PubMed ID)**  Limited jaw opening due to a reduced ability to relax muscles., PMID: 11831344

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
